src/uint_vec/cmp.rs

```diff
@@ -0,0 +1,47 @@
+//! [`UintVec`] comparisons.
+//!
+//! By default these are all constant-time and use the `subtle` crate.
+
+use super::UintVec;
+use crate::Limb;
+use subtle::{Choice, ConstantTimeEq};
+
+impl ConstantTimeEq for UintVec {
+    #[inline]
+    fn ct_eq(&self, other: &Self) -> Choice {
+        let (shorter, longer) = Self::sort_by_precision(self, other);
+        let mut ret = Choice::from(1u8);
+
+        for i in 0..longer.limbs.len() {
+            let a = shorter.limbs.get(i).unwrap_or(&Limb::ZERO);
+            let b = longer.limbs.get(i).unwrap_or(&Limb::ZERO);
```

Comment:
This could probably use a comment that it isn't constant-time with respect to the precision of integers, only their values.